src/CMakeLists.txt

```diff
@@ -221,14 +132,14 @@ endif()
 add_library(libf3d ${F3D_LIBRARY_TYPE}
   ${F3D_SOURCE_FILES}
   ${F3D_HEADER_FILES}
-  ${F3D_READER_FILES}
   ${LIB_PLATFORM_SPECIFIC_FILES}
   )
 
 target_include_directories(libf3d PUBLIC
   ${CMAKE_CURRENT_SOURCE_DIR}
   ${CMAKE_CURRENT_SOURCE_DIR}/readers
   ${CMAKE_CURRENT_BINARY_DIR}
+  ${CMAKE_CURRENT_BINARY_DIR}/readers
```

Comment:
should the readers file added from within the readers CMakeLists ?

Reply:
closing this one, see previous comment